frontend/packages/client/src/components/ProposalCreate/StepOne/index.js

```diff
@@ -460,6 +460,32 @@ const StepOne = ({
             onSelectValue={onSelectStrategy}
             ref={dropDownRef}
           />
+          {defaultValueStrategy && (
+            <>
+              <input
+                type="text"
+                placeholder="Minimum Balance"
+                className="rounded-sm border-light p-3 column is-full mt-4 mb-4"
+                value={stepData?.minBalance}
+                onChange={(event) =>
+                  onDataChange({
+                    minBalance: event.target.value,
+                  })
+                }
+              />
+              <input
+                type="text"
+                placeholder="Maximum Weight"
+                className="rounded-sm border-light p-3 column is-full"
+                value={stepData?.maxWeight}
+                onChange={(event) =>
+                  onDataChange({
+                    maxWeight: event.target.value,
+                  })
+                }
+              />
+            </>
+          )}
```

Comment:
these two fields are not being validated, this means that if the user does not complete them then it will default to the strategy entered values right?

Reply:
yes it's done here: https://github.com/DapperCollectives/CAST/pull/409/files#diff-5d87eb621c5f34cd64b6ae816ecd3b248a750bbff56f7e17c81780b9b52e7808R470

Reply:
Correct!